roles/openstack-meta/defaults/main.yml

```diff
@@ -439,6 +439,14 @@ openstack_meta:
           config_files: /etc/neutron/neutron.conf,/etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini,/etc/neutron/neutron_lbaas.conf
           restart: on-failure
           kill_mode: process
+      f5_oslbaasv2_agent:
```

Comment:
i am assuming you will also enable for rhosp so we consistent. Should we fairly simple and few lines of code.